source/develop/release-management/features/network/ipv6_subnets.html.md

```diff
@@ -100,26 +138,98 @@ The ovirt-provider-ovn provides an opinionated subset of OpenStack's networking
+Just creating the logical router is indeed not enough; each subnet that is
+configured to operate statelessly - either 'slaac' or dhcpv6_stateless -
+requires that very same subnet to be connected to at least one OVN logical router.
 
+That will create OVN logical flows that are responsible for generating the
+Router Advertisements messages the clients require to configure their IP addresses.
+These RA messages are generated as replies to the Router Solicitation messages.
+
+It is also possible to send those periodically, but I see no reason to do so.
 The above change would be implemented in the following way:
+
 1. when a subnet is created, it's **ipv6_address_mode** parameter is stored in
 the subnet's external ids
 2. when that subnet is attached to a router, its **ipv6_ra_configs:address_mode**
 should be set from the subnet's external ids address mode value.
 
+## Guest (VM) configuration
+
+There are three different ways to configure the guests, enabling them to have
+IPv6 addresses, each with different advantages / drawbacks.
+
+### DHCPv6
+
+As indicated on section [](), DHCPv6 is one option to configure the guests. It
+requires **dhclient** to be installed in the guest VM, and since it lacks the
+network prefix length in its message payloads, it will only correctly configure
+guest attached to networks whose prefix length is 64.
+
+This guest configuration is no longer possible in EL8, due to [bug](https://bugs.launchpad.net/ubuntu/+source/isc-dhcp/+bug/1609898).
+
+### Linux Kernel RA configuration
+
+This option does not require any additional package to be installed in the
+guest VM, and involves configuring directy the behavior in the linux kernel,
+as defined [here](https://www.kernel.org/doc/Documentation/networking/ip-sysctl.txt).
+
+This configuration allows for dhcpv6-stateless configurations, where the IPv6 network prefix is sent on RA messages.
+
+To follow it, the following files have to be created, and have a '1' in them:
+- accept_ra
+- autoconf
+- accept_redirects
+- accept_ra_pinfo
+
+And that the *forwarding* flag is disabled - e.g. :
+```bash
+echo "0" > "/proc/sys/net/ipv6/conf/<YOUR_IF_NAME>/forwarding"
+```
+
+This option would work on both EL7 and EL8.
+
+### NetworkManager
+
+This option involves configuring the NetworkManager daemon - which is obviously required.
+
+To configure it, assure the following variables are correctly defined for the
+relevant interface:
+
+```bash
+cat "/etc/sysconfig/network-scripts/ifcfg-<YOUR_IF_NAME>"
+ONBOOT=yes
+IPV6INIT=yes
+IPV6_AUTOCONF=yes
+```
+
+The ifcfg scripts are deprecated on EL8, but still are the backend to Network
+Manager, so, if you need to update the values defined above, the recommended
+way is found below:
+
+```bash
+nm-cli con modify <YOUR_IF_NAME> ipv6.method auto 		# tied to IPV6_AUTOCONF, IPV6INIT
+nm-cli con modify <YOUR_IF_NAME> connection.autoconnect yes 	# tied to ONBOOT
+```
+
+This option would work on both EL7 and EL8.
+
 ## User flows
```

Comment:
The flows would be more useful, if they cover all steps:
1. Create network in ManageIQ, in another flow in oVirt Engine
2. Create subnet in ManageIQ/oVirt Engine
3. The subnet has to be connected to a router
3. Assign related vNIC profile to a vNIC of a VM
4. what should happen inside the VM for RHEL 8 default, or with modified config

Reply:
+1 on this

Reply:
I don't get what you expect with 'flow' - what's the format?... diagrams? words? lists of stuff to do?..

Reply:
I've gone with lists. 

Let me know if that's OK.